hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -220,6 +220,7 @@ private void rolloverIfNeeded() throws IOException {
       LOG.info("CurrentSize " + getCurrentSize() + " has reached threshold " + sizeThreshold
           + ". Rolling over to the next version");
       rollOver();
+      createNewFile();
     }
```

Comment:
We better not generate a new physical file handle when rollover, the good strategy is: only generate a new file handle when we want to write real data set.